stream-handler/handlers/apply_user_filter_handler.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+
+class ApplyUserFilterHandler:
+
+    def __init__(self):
+        self.log = logging.getLogger()
```

Comment:
Let's do `getLogger(__name__)` in our modules so that we can set different levels for our code and third-party libs. I often find third-party libs way too verbose at DEBUG level.